src/patternfly/components/Wizard/wizard.scss

```
@@ -478,6 +509,26 @@
   flex-grow: 1;
 }
 
+.#{$wizard}__nav-link-main {
+  display: flex;
+  flex-grow: 1;
+  justify-content: space-between;
+  padding-block-start: var(--#{$wizard}__nav-link--PaddingBlockStart);
+  padding-block-end: var(--#{$wizard}__nav-link--PaddingBlockEnd);
+  padding-inline-start: var(--#{$wizard}__nav-link--PaddingInlineStart);
+  padding-inline-end: var(--#{$wizard}__nav-link--PaddingInlineEnd);
+  background-color: var(--#{$wizard}__nav-link--BackgroundColor);
+  border: none;
+  border-radius: var(--#{$wizard}__nav-link--BorderRadius);
```

should we rename these?

```suggestion
  padding-block-start: var(--#{$wizard}__nav-link-main--PaddingBlockStart);
  padding-block-end: var(--#{$wizard}__nav-link-main--PaddingBlockEnd);
  padding-inline-start: var(--#{$wizard}__nav-link-main--PaddingInlineStart);
  padding-inline-end: var(--#{$wizard}__nav-link-main--PaddingInlineEnd);
  background-color: var(--#{$wizard}__nav-link-main--BackgroundColor);
  border: none;
  border-radius: var(--#{$wizard}__nav-link-main--BorderRadius);
```